app/views/content/shared/quotes/_quote-teaching-abigail.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="row">
+  <div class="col-space-l col-space-l-top">
+    <%= render Content::QuoteWithImageComponent.new(
+      title: "Teacher career progresion stories",
+      text: "<p>I completed the National Professional Qualification for Senior Leadership (NPQSL), which has involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p><a href='/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story'>Read Abigail's career progression story</a>",
```

Comment:
```suggestion
      text: "<p>I completed the National Professional Qualification for Senior Leadership (NPQSL), which involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p><a href='/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story'>Read Abigail's career progression story</a>",
```
Suggested change (ignore as needed) plus do we need a full-stop after the link text?

Reply:
```suggestion
      text: "<p>I completed the National Professional Qualification for Senior Leadership (NPQSL), which involved running a project across the whole school. This has given me the opportunity to work more closely with the senior leadership team and has helped strengthen my relationships with colleagues in other departments in the school.</p><p><a href='/life-as-a-teacher/teaching-as-a-career/abigails-career-progression-story'>Read Abigail's career progression story</a>".</p>,
```